appendices/assignments.md

```diff
@@ -0,0 +1,197 @@
+    to Pull Requests.
+
+## Third project: Each group creating a package
+
+This assignment is *optional* as it depends on multiple factors like class size,
+duration of course, or virtual vs in-person. This would be the third and final 
+deliverable for the learners. There are no final exercises in each chapter that
+learners need to complete related to this assignment. Exercises and the
+structure of the group work would be determined by the instructor.
+This assignment would best be due sometime after the teamwork lesson (REF).
+
+- Using the same process as was done with the first project assignment,
+complete a package as a team, using the tools taught in the team and community
+chapters (TODO: REF).
+
+## Rubric and expectations of the learner
+
+TODO: Can fill this out more as we work on book
+
+The expectations for completing the project assignments change as the course 
+progresses.
+
+- After the first Git lesson (REF), project should have frequent and consistent
+commits and, after the GitHub lesson (REF), pushes to GitHub.
+The GitHub history will be checked to confirm this happens.
+- Tasks in the final chapter exercises should be completed shortly after
+finishing the chapter.
+
+## For instructors (add once file is merged)
+
+Making an interesting project assignment can be quite difficult, so this section
+gives some suggestions and guidance on how what the specific project should be.
+The specific type and number of assignments are of course ultimately decided by
+the instructor of the course, but at the least, this course is designed to with
+two main project assignments in mind, as described above. The specific details of
+the assignments are completed by the instructor, however, this section contains
+some general advice and guidelines.
+
+- The first project package description already provides a suggested package to
+create. As the instructor you can modify this example to focus on weather
+from a specific city, a specific time period, or different aspects of the weather
+(e.g. temperature only).
+- While the project package does not necessarily need to involve data, it might
+be more motivating for the learners to work on if the project did include data of 
+some time. R is a language predominantly used by researchers and data scientists
+for doing data analysis, visualization, and related reporting. Given this,
+learners taking this course are probably already familiar with these tasks and
+would benefit more from the course if they see and use an application they are
+familiar with.
+    - If it doesn't involve data, it's still recommended to structure the project
+    package around using or working with data in some way.
+- Package problems are always better if they are targeted to the background 
+knowledge of the instructor and the learners. Field-specific and class
+context-dependent problems will more likely be remembers and learned than other
+problems.
```

Comment:
A re-ordering and some minor wording edits:
```suggestion
- Package problems are always better if they are targeted to the background 
knowledge of the instructor and the learners. Field-specific and class
context-dependent problems will more likely be remembered and learned than other
problems.
- In the final exercises, we provide a suggested suggested package - 
weather summaries, as described above. 
As the instructor, one way to tailor your class, 
is to modify this example to focus on weather from a local city, 
a specific time period, or different aspects of the weather (e.g. temperature only).
- While the project package does not necessarily need to involve data, it might
be more motivating for the learners to work on if the project does include data. 
R is a language predominantly used by researchers and data scientists
for doing data analysis, visualization, and related reporting. Given this,
learners taking this course are probably already familiar with these tasks and
would benefit more from the course if they see and use an application they are
familiar with.
    - If it doesn't involve data, it's still recommended to structure the project
    package around using or working with data in some way.

```